Tools/Scripts/hooks/prepare-commit-msg

```diff
@@ -34,6 +35,7 @@ UNPACK_SECONDARY_RE = re.compile(r' \(({})\)'.format(COMMIT_REF_BASE))
 sys.path.append(SCRIPTS)
 from webkitpy.common.checkout.diff_parser import DiffParser
 from webkitbugspy import radar
+from webkitscmpy.commit_parser import CommitMessageParser
```

Comment:
This will inevitably break the hook if you're on an older branch.

Reply:
Indeed! `DiffParser` isn't so bad because it's ancient.

What is our commit message template behavior in if we fail this import?

Reply:
Based on our discussion, I've implemented a fallback to the original commit message template which doesn't update the change logs. It also only tries to import the parser if we're amending and `update_changelog = True` to minimize impact